squall/responses.py

```diff
@@ -74,16 +77,35 @@ def __init__(
         self.raw_headers = raw_headers = init_headers(
             body, self.charset, self.media_type, headers
         )
-        self.send_start = {
+        self.send_start: Dict[str, Any] = {
             "type": "http.response.start",
             "status": status_code,
             "headers": raw_headers,
         }
         self.send_body = {"type": "http.response.body", "body": body}
 
     async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
+        compression: Optional[Compression] = scope["app"].compression
+        send_body: Dict[str, Any] = self.send_body
```

Comment:
```
body = self.send_body.get('body', '')
```
Seems like there are we are saving some attributes accessing, but do more dictionary lookups. So it's like  vice-versa, but less readable

Reply:
changed